src/mds/PurgeQueue.h

```diff
@@ -88,7 +88,7 @@ enum {
   // How many items have been finished by PurgeQueue
   l_pq_executing_ops,
   l_pq_executing_ops_high_water,
-  l_pq_executing,
+  l_pq_executing_items,
```

Comment:
Do we really need to change this? The perf counters are (mostly) used by devs as a sneak peak into how the MDS is churning through some of its tasks. While it's good to do away with weird/obscure naming, I don't think this perf counter fits in that category.

Reply:
This is confusing for even the devs since it's quite confusing at first to understand what is it tracking. This came up after https://bugzilla.redhat.com/show_bug.cgi?id=2314327 where the support/customer were facing some struggle figuring out the differencen b/w `pq_executing` and `pq_executing_ops`. I feel this is a minor QoL change which would help understand there counters better.